frontend/templates/index.html

```diff
@@ -1,26 +1,70 @@
 <html>
 
 <head>
-    Hadith Correction App
-    <script src="https://cdn.jsdelivr.net/npm/vue"></script>
+  Hadith Correction App
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">

```

Comment:
how is this sha calculated?

Reply:
This is taken from the [bootstrap 4 docs](https://getbootstrap.com/docs/4.5/getting-started/introduction/).